modules/react/skeleton/index.ts

```diff
@@ -1,11 +1 @@
-import Skeleton from './lib/skeleton';
-import SkeletonText from './lib/parts/skeletonText';
-import SkeletonHeader from './lib/parts/skeletonHeader';
-import SkeletonShape from './lib/parts/skeletonShape';
-
-export {Skeleton, SkeletonText, SkeletonHeader, SkeletonShape};
-
-export default Skeleton;
-export * from './lib/parts/skeletonHeader';
-export * from './lib/parts/skeletonShape';
-export * from './lib/parts/skeletonText';
+export * from './lib/skeleton';
```

Comment:
Breaking.

Reply:
@willklein You'll need to update the v5 Migration Guide given the new compound component API for Skeleton. I'd add a `Skeleton` section under `Component Changes`. The [Card section](https://github.com/Workday/canvas-kit/blob/prerelease/v5/modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx#card) should be a good reference for how to organize your content.